cmd/gen-release-notes/main.go

```diff
@@ -85,20 +87,31 @@ func main() {
 			os.Exit(1)
 		}
 
-		if err := writeAsMarkdown(filename, output); err != nil {
+		if err := createDirIfNotExists(outDir); err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to create our dir: %s\n", err.Error())
+		}
+		if err := writeAsMarkdown(path.Join(outDir, filename), output); err != nil {
 			fmt.Fprintf(os.Stderr, "Failed to write markdown: %s\n", err.Error())
 		} else {
 			fmt.Printf("Wrote markdown to %s\n", filename)
 		}
 
-		if err := writeAsHTML(filename, output); err != nil {
+		if err := writeAsHTML(path.Join(outDir, filename), output); err != nil {
 			fmt.Fprintf(os.Stderr, "Failed to write HTML: %s\n", err.Error())
 		} else {
 			fmt.Printf("Wrote markdown to %s.html\n", filename)
 		}
 	}
 }
 
+func createDirIfNotExists(path string) error {
+	err := os.Mkdir(path, 0755)
+	if err == nil || os.IsExist(err) {
```

Comment:
You can remove the `err == nil` from above

Reply:
Good catch and done. Thanks